web/src/layout/leftpanel/countrypanel.jsx

```diff
@@ -323,7 +324,9 @@ const CountryPanel = ({ electricityMixMode, isMobile, tableDisplayEmissions, zon
       <CountryPanelWrap>
         {hasParser ? (
           <React.Fragment>
-            <BySource>{__('country-panel.bysource')}</BySource>
+            <BySource>
+              {__(timeAggregate != TIME.HOURLY ? 'country-panel.averagebysource' : 'country-panel.bysource')}
```

Comment:
```suggestion
              {__(timeAggregate !== TIME.HOURLY ? 'country-panel.averagebysource' : 'country-panel.bysource')}
```

Reply:
I dont think it changes functionality in any case but I just find that odd things happens when using !=